### PR TITLE
Feat/system tests on windows

### PR DESCRIPTION
**Pull request by mhaas**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,10 +20,15 @@ notifications:
     if: branch = master AND type != pull_request
     on_failure: always
     recipients:
-      michael.haas01@sap.com
-      karthik.muthuswamy@sap.com
-      francesco.alda@sap.com
-      evgeny.arnautov@sap.com
+      # Michael
```
Comment (mhaas) [inline .travis.yml:23]: @EvgenyArn This should be better!

**Comment by mhaas** [comment]
Screenshot 2020-07-02 at 10 14 26

System tests working fine on Windows.

File: .travis.yml (hunk 1 above)
Comment (karthikmswamy) [reply .travis.yml:23]: What's this? And, is this a hash of the email address?

Comment (karthikmswamy) [reply .travis.yml:23]: My question is more out of curiosity than a question :)

Comment (EvgenyArn) [reply .travis.yml:23]: @mhaas looks fancy, what is it? :)

Comment (mhaas) [reply .travis.yml:23]: @EvgenyArn In Travis, each repo has an associated public and private key pair. I can encrypt data with the public key, and  Travis can decrypt it with the private key. This is also how credentials are handled for pypi upload and for the DAR service key used for the system tests :)

**Pull request merged**